src/main.rs

```diff
@@ -86,6 +86,9 @@ struct Args {
     /// Use `/collections/<collection-name>/snapshots/recover` API instead.
     #[arg(long, value_name = "PATH")]
     storage_snapshot: Option<String>,
+
+    #[arg(long)]
+    mmap_random: bool,
```

Comment:
@generall  Do we have any use cases where mmap random should be disabled?

Reply:
Also, I guess random access should be the default behavior. I propose to add `mmap_linear` flag instead of `mmap_random`.

Reply:
The plan is to use `Advice::Random` by default, but allow changing it to `Advice::Normal` through config file.

BTW, @generall, do we want to allow setting any _other_ `Advice` (e.g., other than `Normal` and `Random`)?

Reply:
if it is not too complicated, let's add others as well

Reply:
regarding `mmap_random` flag, we don't need in CLI